README.md

```diff
@@ -1,34 +1,26 @@
+- [Final](https://eips.ethereum.org/all) and [Active](https://eips.ethereum.org/all) proposals that are recorded.
+- The [EIP process](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md#eip-work-flow) that governs the EIP repository.
 
-Your first PR should be a first draft of the final EIP. It must meet the formatting criteria enforced by the build (largely, correct metadata in the header). An editor will manually review the first PR for a new EIP and assign it a number before merging it. Make sure you include a `discussions-to` header with the URL to a discussion forum or open GitHub issue where people can discuss the EIP as a whole.
+Achieving "Final" status in this repository only represents that a proposal has been reviewed for technical accuracy. It is solely the responsibilty of the implementer to decide if these proposals will be useful to you and if they represent best practice.
 
-If your EIP requires images, the image files should be included in a subdirectory of the `assets` folder for that EIP as follow: `assets/eip-X` (for eip **X**). When linking to an image in the EIP, use relative links such as `../assets/eip-X/image.png`.
+## Project Goal
 
-Once your first PR is merged, we have a bot that helps out by automatically merging PRs to draft EIPs. For this to work, it has to be able to tell that you own the draft being edited. Make sure that the 'author' line of your EIP contains either your Github username or your email address inside <triangular brackets>. If you use your email address, that address must be the one publicly shown on [your GitHub profile](https://github.com/settings/profile).
+The Ethereum Improvement Proposals repository exists as a place to discuss concrete proposals with potential users of the proposal and the Ethereum community at large.
 
-When you believe your EIP is mature and ready to progress past the draft phase, you should do one of two things:
+We promote high-quality, peer-reviewed proposals in two categories:
 
- - **For a Standards Track EIP of type Core**, ask to have your issue added to [the agenda of an upcoming All Core Devs meeting](https://github.com/ethereum/pm/issues), where it can be discussed for inclusion in a future hard fork. If implementers agree to include it, the EIP editors will update the state of your EIP to 'Accepted'.
- - **For all other EIPs**, open a PR changing the state of your EIP to 'Final'. An editor will review your draft and ask if anyone objects to its being finalised. If the editor decides there is no rough consensus - for instance, because contributors point out significant issues with the EIP - they may close the PR and request that you fix the issues in the draft before trying again.
+- [Consensus Changes](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md#eip-types) require people running an Ethereum client to update their software to implement the change. If enough people do not upgrade then an existing network will fork into two separate networks.
```

Comment:
Links should be relative, not absolute.  That way they follow in forks (e.g., when creating a PR).

Reply:
Thank you, fixed in 879ebdd9e45bb9d874d053f673230ed8f0ccd849